build/build.xml

```diff
@@ -367,7 +367,7 @@
 		<build-addon name="help_fr_FR" />
 		<build-addon name="help_ja_JP" />
 		<build-addon name="highlighter" />
-		<build-addon name="httpsInfo" />
+		<build-addon name="httpsinfo" />
```

Comment:
Does this change work? I thought it had to be the name of the package, which is still httpsInfo :/
Ditto the change below.